### ISSUE TITLE
Doubt in transfer learning parameters tutorial.

### ISSUE THREAD

**Issue by THEFASHIONGEEK** [Tessellate-Imaging/monk_v1#21]
![image](https://user-images.githubusercontent.com/50513357/75762297-720e3a00-5d60-11ea-98d7-85d2332b038e.png)

1. Why there is a difference in potentially trainable layers and actual trainable layers even after we have unfreezed the base network (desnenet121) using mxnet as shown in the above image?

![image](https://user-images.githubusercontent.com/50513357/75762653-0bd5e700-5d61-11ea-9807-e669f75971fd.png)

2.Why we have different no.of layers in PyTorch and mxnet for the same densenet121?
3. Why are Unfreezed base network in PyTorch as same no of potentially trainable layers and actual trainable layers if the first image is correct?

**Comment by abhi-kumar** [comment]
Thats a very good doubt you have raised. 

It was a bug, that was considering number of parameters to be trained a s number of layers. But that's not always the case, for example, in convolution layer there are 2 named parameters - weight and bias, so wherever this layer has no bias it was counting it as one, but when it had bias as in pytorch, it counted a single layer as 2.

Same was the case with dense and normalization layers.

The next issue was with activations, keras backed counted it as a trainable layer whereas the other backends didn't. Now only the layers that can actually be trained are marked as "trainable layers".

The notebooks too have been updated.

**Comment by abhi-kumar** [comment]
Closing the issue.
